README.md

```diff
@@ -165,6 +165,7 @@ A valid configuration file should look like this:
 ```properties
     # Bridge configuration
     bridge.id=my-bridge
+    bridge.default.topic=default_topic
```

Comment:
```suggestion
    bridge.topic.default=default_topic
```

I think this should be the other way around so `bridge.topic.default` just in case we would like to set more properties about this topic in the future.

Reply:
This makes total sense.